src/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
- [ ]  There is an issue, I can not compile the app in the browser, kindy solve this :arrow_down: 

![err](https://user-images.githubusercontent.com/57562869/155017624-5d62f9a1-fe8e-40f5-8a94-08e5447fe72b.png)